### PR TITLE
add: `cmake-data-deb`

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

File: packages/cmake-data-deb/cmake-data-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name="cmake-data-deb"
+gives="cmake-data"
+version="3.25.1"
+url="http://archive.ubuntu.com/ubuntu/pool/main/c/cmake/cmake-data_${version}-1_amd64.deb"
+description="CMake data files (modules, templates and documentation)"
+hash=""
```
Comment (Henryws) [inline packages/cmake-data-deb/cmake-data-deb.pacscript:6]: Hash can't be empty.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/cmake-data-deb/cmake-data-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name="cmake-data-deb"
+gives="cmake-data"
+version="3.25.1"
+url="http://archive.ubuntu.com/ubuntu/pool/main/c/cmake/cmake-data_${version}-1_amd64.deb"
+description="CMake data files (modules, templates and documentation)"
+hash="aaa"
```
Comment (Henryws) [inline packages/cmake-data-deb/cmake-data-deb.pacscript:6]: This.. also won't work. Fill it correctly.

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
You don't have to force push every time btw.

**Comment by Zahrun** [comment]
> You don't have to force push every time btw.

I thought we need to keep only one commit per pr

**Review by Henryws — CHANGES_REQUESTED** [review]
Add `cmake-deb` to packagelist

**Comment by Zahrun** [comment]
Arf I mixed 2 branches, just one second

**Comment by Henryws** [comment]
> > You don't have to force push every time btw.
> 
> I thought we need to keep only one commit per pr

No, we squash merge the commit anyways, so it's *merged* as one commit, but you can work on multiple in this PR.

**Comment by Zahrun** [comment]
> > > You don't have to force push every time btw.
> > 
> > 
> > I thought we need to keep only one commit per pr
> 
> No, we squash merge the commit anyways, so it's _merged_ as one commit, but you can work on multiple in this PR.

Ah ok, that's nice

File: packages/cmake-data-deb/cmake-data-deb.pacscript [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name="cmake-data-deb"
+gives="cmake-data"
+version="3.25.1"
+description="CMake data files (modules, templates and documentation)"
+case ${codename} in
```
Comment (Henryws) [inline packages/cmake-data-deb/cmake-data-deb.pacscript:6]: Where's this being set?

File: packages/cmake-data-deb/cmake-data-deb.pacscript [hunk 4]
```diff
@@ -0,0 +1,17 @@
+name="cmake-data-deb"
+gives="cmake-data"
+version="3.25.1"
+description="CMake data files (modules, templates and documentation)"
+codename="$(lsb_release -cs)"
+case ${codename} in
```
Comment (Henryws) [inline packages/cmake-data-deb/cmake-data-deb.pacscript:6]:
```suggestion
case "${codename}" in
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**